File: LDCoreData-OC/ManEntity+CoreDataProperties.m
//
//  ManEntity+CoreDataProperties.m
//  LDCoreData-OC
//
//  Created by LiuQiqiang on 2018/4/21.
//  Copyright © 2018年 LiuQiqiang. All rights reserved.
//
//

#import "ManEntity+CoreDataProperties.h"

@implementation ManEntity (CoreDataProperties)

+ (NSFetchRequest<ManEntity *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"ManEntity"];
}

@dynamic height;
@dynamic name;
@dynamic weight;

@end

File: LDCoreDataTool-Swift/ManEntity+CoreDataProperties.swift
//
//  ManEntity+CoreDataProperties.swift
//  LDCoreDataTool2-Swift
//
//  Created by Artron_LQQ on 2017/7/3.
//  Copyright © 2017年 Artup. All rights reserved.
//

import Foundation
import CoreData


extension ManEntity {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<ManEntity> {
        return NSFetchRequest<ManEntity>(entityName: "ManEntity")
    }

    @NSManaged public var height: Float
    @NSManaged public var name: String?
    @NSManaged public var weight: Float

}
